fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/SearchFilter.java

```diff
@@ -42,6 +43,22 @@
         QueryType(String type) {
             this.type = type;
         }
+
+        /**
+         * Get the QueryType enum from it's search type.
+         *
+         * @param type  The type of query (i.e. "insensitive_contains")
```

Comment:
`(i.e. "insensitive_contains")` indicates this is the only QueryType? I would remove it.